CHANGES.md

```diff
@@ -1,504 +1,600 @@
 # pyfakefs Release Notes
+
```

Comment:
These are a lot of changes - did you do some autoformatting here?

Reply:
Ok, I see that there are no real changes (e.g. the output won't change), so I will ignore this...

Reply:
I reverted the unrelated changes before seeing your second comment.